angular/test/test-app/src/main.ts

```diff
@@ -8,5 +8,8 @@ if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule)
-  .catch(err => console.error(err));
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
starter from ngCLI does not do that, why us?

Reply:
This is for SSR. To make sure that the DOM is ready before bootstrapping the client side app.